mozfun/bits28/days_since_seen/udf.sql

```diff
@@ -15,7 +15,7 @@ CREATE OR REPLACE FUNCTION bits28.days_since_seen(bits INT64) AS (
 );
 
 SELECT
-  assert_null(udf.bits28_days_since_seen(0)),
+  assert_null(bits28.days_since_seen(0)),
```

Comment:
this was testing the function from the old location, probably copy-pasta from when this udf was migrated.